### PR TITLE
Destroy the correct service from the inspector

### PR DESCRIPTION
**Pull request by huwshimi**
Fixes: #3565.

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- add two apps
- click on one, then click on the other
- click Destroy in the inspector
- the second app you clicked on should get removed

**Comment by huwshimi** [comment]
[Destroy the correct service from the inspector](https://trello.com/c/vL7eaWeR/385-destroy-the-correct-service-from-the-inspector)

File: jujugui/static/gui/src/app/components/inspector/service-overview/service-overview.js [hunk 1]
```diff
@@ -251,7 +251,7 @@ class ServiceOverview extends React.Component {
   _destroyService() {
     // db, env, and service have already been bound to this function in
```
Comment (frankban) [inline jujugui/static/gui/src/app/components/inspector/service-overview/service-overview.js]: So it seems that is comment must be updated as well now.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8564/

File: jujugui/static/gui/src/app/components/inspector/inspector.js [hunk 2]
```diff
@@ -72,19 +72,19 @@ class Inspector extends React.Component {
...
             changeState={changeState}
-            charm={this.props.charm}
-            clearState={this.props.clearState}
-            destroyService={this.props.destroyService}
-            displayPlans={this.props.displayPlans}
-            getUnitStatusCounts={this.props.getUnitStatusCounts}
-            modelUUID={this.props.modelUUID}
+            charm={nextProps.charm}
+            clearState={nextProps.clearState}
+            destroyService={nextProps.destroyService}
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/components/inspector/inspector.js]: This was generating the state using the previous props, instead of the new props.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8565/

**Comment by huwshimi** [comment]
Thanks to you @frankban! :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Build failed: Attempt to land pull request failed
        build url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge/2733

**Comment by huwshimi** [comment]
:shipit:

**Pull request merged**